astroid/tests/unittest_brain.py

```diff
@@ -50,6 +50,13 @@
     HAS_PYTEST = True
 except ImportError:
     HAS_PYTEST = False
+
+try:
+    import attr as attr_module # pylint: disable=unused-import
```

Comment:
We should expose this in tox so that the tests could run in CI: https://github.com/PyCQA/astroid/blob/be694f811f7df89c4fcc672ef09c0e1fc499cf23/tox.ini#L14